packages/airnode-protocol/contracts/rrp/authorizers/admin/interfaces/IERC20Extended.sol

```diff
@@ -0,0 +1,6 @@
+// SPDX-License-Identifier: MIT
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+
+interface IERC20Extended is IERC20 {
```

Comment:
I thought this was not needed anymore because of the openzepplin meta contract?

Reply:
I probably deleted the implementation and forgot to delete the interface. Good catch.